flights_nyc13/pre_processing.R

```diff
@@ -0,0 +1,33 @@
+###20.08
+ggplot(flights) +
+  aes(x = "", y = dep_delay) +
+  geom_boxplot(fill = "#0c4c8a") +
+  theme_minimal()
+
+boxplot.stats(flights$dep_delay)$out
+
+#merged flights + weather + planes
+combined_df<-merge(flights, planes, by="tailnum")
```

Comment:
suggestion:
change variable name to a name that includes which datasets are combined
for example - fwp_data (flights,weather,planes)